blocks/components/editable/style.scss

```diff
@@ -10,4 +10,15 @@
 	&:focus {
 		outline: none;
 	}
+
+	&.is-empty {
+		position: relative;
+	}
+
+	&.is-empty:before {
```

Comment:
Should the `::placeholder` pseudo element be used here?  The browser support is not good so `::before` is probably best?